### ISSUE TITLE
Example of Delegated Inception and Delegated Rotation?

### ISSUE THREAD

**Issue by FrankC01** [THCLab/keriox#20]
Moved from https://github.com/WebOfTrust/keriox/issues/5

Looking at the EventMsgBuilder it is not clear for Dip what the delegator field is to be set to?

Is it the IdentifierPrefix for the delegator which previously had been Incepted? In other words, do I take the IdentifierPrefix of the result of inception for the Delegator keys and use at the delegator for the keys I'm Dip'ing so to speak?

**Comment by edytapawlak** [comment]
Yes, the delegator field is IdentifierPrefix of the previously incepted delegator. I added the [test](https://github.com/THCLab/keriox/blob/e8471fbf93c15f8846bd3b1e1212eb04bd700a2c/keriox_core/src/processor/tests.rs#L232) for processing delegated events in #21 . It doesn`t use  EventMsgBuilder directly, but it may be helpful.

**Comment by blelump** [comment]
Closing for now. Feel free to reopen if needed.
